gw_spaceheat/layout_gen/multi.py

```diff
@@ -19,8 +19,8 @@
 
 class SensorNodeGenCfg(BaseModel):
     NodeAlias: str
-    Role: Role
     DisplayName: str
+    Role: "Role" = Role.Unknown
```

Comment:
because Roles are going to get phased out anyway - they are too generic to be used for determining what code runs. The mechanism we have in place right now for determining the driver is still useful and can be used to determine the code for "leaf" spaceheat nodes that are either purely sensing or actuating, but nodes that perform some more specific appliance-level function (for example, running a thermostat or determining aquastatic control of a buffer tank) will need code determined very specifically by the "House 0" design scheme and then directly by the ShNode names.